@kiva/kv-components/vue/KvMap.vue

```diff
@@ -408,6 +505,96 @@ export default {
 				}, timeout);
 			});
 		},
+		getCountriesData() {
+			const countriesFeatures = countriesBorders.features ?? [];
+
+			countriesFeatures.forEach((country, index) => {
+				const countryData = this.countriesData.find((data) => data.isoCode === country.properties.ISO_A2);
+				if (countryData) {
+					countriesFeatures[index].lenderLoans = countryData.value;
+					countriesFeatures[index].numLoansFundraising = countryData.numLoansFundraising;
+				}
+			});
+
+			return countriesBorders;
+		},
+		countryStyle(feature) {
+			return {
+				color: kvTokensPrimitives.colors.white,
+				fillColor: this.getCountryColor(feature.lenderLoans),
+				weight: 1,
+				fillOpacity: 1,
+			};
+		},
+		getCountryColor(lenderLoans) {
+			const loanCountsArray = [];
+			this.countriesData.forEach((country) => {
+				loanCountsArray.push(country.value);
+			});
+
+			const maxNumLoansToOneCountry = Math.max(...loanCountsArray);
+			const intervals = getLoansIntervals(1, maxNumLoansToOneCountry, 6);
+
+			if (intervals.length === 1) {
+				const [inf, sup] = intervals[0]; // eslint-disable-line no-unused-vars
+
+				for (let i = 0; i < sup; i += 1) {
+					const loansNumber = i + 1;
+
+					if (lenderLoans && lenderLoans >= loansNumber && lenderLoans < loansNumber + 1) {
+						return kvTokensPrimitives.colors.brand[this.mapColors[i]];
+					}
+				}
+			} else {
+				for (let i = 0; i < intervals.length; i += 1) {
+					const [inf, sup] = intervals[i];
+					if (lenderLoans && lenderLoans >= inf && lenderLoans <= sup) {
+						return kvTokensPrimitives.colors.brand[this.mapColors[i]];
+					}
+				}
+			}
+
+			return kvTokensPrimitives.colors.gray[300];
+		},
+		onEachCountryFeature(feature, layer) {
+			const loansString = feature.lenderLoans
+				? `${feature.lenderLoans} loan${feature.lenderLoans > 1 ? 's' : ''}`
+				: '0 loans';
+			const countryString = `${feature.properties.NAME} <br/> ${loansString}`;
+
+			layer.bindTooltip(countryString, {
+				sticky: true,
+			});
+
+			layer.on({
+				mouseover: this.highlightFeature,
+				mouseout: this.resetHighlight,
+			});
+		},
+		highlightFeature(e) {
+			const layer = e.target;
+
+			layer.setStyle({
+				fillColor: kvTokensPrimitives.colors.gray[500],
+			});
+		},
+		resetHighlight(e) {
+			const layer = e.target;
+			const { feature } = layer;
+
+			layer.setStyle({
+				fillColor: this.getCountryColor(feature.lenderLoans),
+			});
+		},
+		circleMapClicked(countryIso) {
+			this.$emit('country-lend-filter', countryIso);
+			this.$router.push({
```

Comment:
Given that the router isn't defined in the component library, this click method should probably just emit.